client/gutenberg/extensions/mailchimp/edit.jsx

```diff
@@ -144,12 +144,13 @@ class MailchimpSubscribeEdit extends Component {
 						'You need to connect your Mailchimp account and choose a list in order to start collecting Email subscribers.'
 					) }
 					<br />
-					<ExternalLink href={ connectURL }>{ __( 'Set up Mailchimp form' ) }</ExternalLink>
 					<br />
-					<br />
-					<Button isDefault onClick={ this.apiCall }>
-						{ __( 'Re-check Connection' ) }
+					<Button isPrimary href={ connectURL } target="_blank">
```

Comment:
Can we use `isDefault isLarge` rather than `isPrimary`?

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/31089/commits/7f06874875170f7863410d53bbc40f29f69ce847